docs/DeveloperGuide.md

```diff
@@ -460,7 +476,12 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
       Steps 1a1 to 1a2 are repeated until the data entered are correct.              
       Use case resumes from Step 2.
 
-**Use Case: UC04 - Add lesson for student**  
+* **1b**. No contacts found matching the entered keyword.
+    * **1b1**. TutorEase displays a message: "No contacts found with the given keyword(s)."  
+    * **1b2**. Tutor enters a new keyword or cancels the search.  
+      Use case resumes from Step 2 or ends if canceled.  
```

Comment:
I think it should be cancelled instead of canceled if we are following british english.